recipes/coverage-lcov/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "coverage-lcov" %}
+{% set version = "0.2.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d65a77e4694c5dab041b2ba9725d819de12d8f26efc93f47bcd3dc25a74f017c
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - coverage-lcov = coverage_lcov.cli:main
+
+requirements:
+  host:
+    - pip
+    - python >=3.6.3
+    - poetry-core >=1.0.0
+  run:
+    - python >=3.6.3
+    - toml >=0.5
+    - types-toml <0.2.0,>=0.1.5
```

Comment:
Are you sure that these two are required?

Upstream lists coverage with the toml extra and coverage specifies the toml extra as tomli: https://github.com/nedbat/coveragepy/blob/a0c7ee45371778ac71abd1ce90a79113d4819d56/setup.py#L111

tomli is already included in the conda-forge package: https://github.com/conda-forge/coverage-feedstock/blob/b9288b50f583f8035b3dab7f6b96155ce33501d2/recipe/meta.yaml#L28

Reply:
Seems like tomli it's not being pulled for some reason:

```
import: 'coverage_lcov'
Traceback (most recent call last):
  File "/home/conda/staged-recipes/build_artifacts/coverage-lcov_1648591978751/test_tmp/run_test.py", line 2, in <module>
    import coverage_lcov
  File "/home/conda/staged-recipes/build_artifacts/coverage-lcov_1648591978751/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl/lib/python3.9/site-packages/coverage_lcov/__init__.py", line 3, in <module>
    import toml
ModuleNotFoundError: No module named 'toml'
Tests failed for coverage-lcov-0.2.4-pyhd8ed1ab_0.tar.bz2 - moving package to /home/conda/staged-recipes/build_artifacts/broken
WARNING:conda_build.build:Tests failed for coverage-lcov-0.2.4-pyhd8ed1ab_0.tar.bz2 - moving package to /home/conda/staged-recipes/build_artifacts/broken
TESTS FAILED: coverage-lcov-0.2.4-pyhd8ed1ab_0.tar.bz2
##[error]Bash exited with code '1'.
Finishing: Run docker build
```

Reply:
I thought I had seen something related in the dependent package that has a new dependency on this one, and I remembered where. Looks like it could be related:

https://github.com/meejah/txtorcon/blob/9d631c9c6c72a570d80159a8d4e348ba4564ba8e/tox.ini#L41

Reply:
Ahh, you‘re right. Then it’s better to declare it here. Can you please just add a note to the meta.yaml that this is added because it’s missing somewhere else so this can be revisited in the future?

Reply:
Sure thing, that's done.